pkgs/development/compilers/mono/generic.nix

```diff
@@ -20,9 +22,6 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = if stdenv.isDarwin then "" else "-lgcc_s" ;
 
-  # To overcome the bug https://bugzilla.novell.com/show_bug.cgi?id=644723
-  dontDisableStatic = true;
```

Comment:
Won't this impact previous (<6) version of mono?

Reply:
I did nix-review and all compiled and what I tried to run worked well, or at least no worse than before. Notice the bugzilla is quite ancient. I removed it based on suggestion in https://github.com/NixOS/nixpkgs/pull/58797#issuecomment-489521629.

Reply:
Ok then.